src/openrct2-ui/CursorRepository.cpp

```diff
@@ -14,41 +14,27 @@
 *****************************************************************************/
 #pragma endregion
 
+#include <cmath>
+
 #include <openrct2/common.h>
-#include <SDL.h>
```

Comment:
Why did you remove this include?

Reply:
I moved it to `CursorRepository.h` because I need the SDL_Cursor definition there.